content/docs/jsx-in-depth.md

```diff
@@ -196,29 +196,29 @@ function NumberDescriber(props) {
 }
 ```
 
-You can learn more about [conditional rendering](/docs/conditional-rendering.html) and [loops](/docs/lists-and-keys.html) in the corresponding sections.
+Anda bisa mempelajari lebuh lanjut [proses *render* kondisional](/docs/conditional-rendering.html) dan [pengulangan](/docs/lists-and-keys.html) di bagian-bagian yang sesuai.
 
-### String Literals {#string-literals}
+### *String Literal* {#string-literals}
 
-You can pass a string literal as a prop. These two JSX expressions are equivalent:
+Anda dapat mengoper sebuah *string literal* sebagai sebuah *prop*. Kedua ekspresi JSX berikut ini sama:
 
 ```js
 <MyComponent message="hello world" />
 
 <MyComponent message={'hello world'} />
 ```
 
-When you pass a string literal, its value is HTML-unescaped. So these two JSX expressions are equivalent:
+Ketika Anda mengoper sebuah *string literal*, nilainya tidak luput dari HTML (*HTML-unescaped*). Jadi kedua eskpresi JSX berikut ini sama:
 
 ```js
 <MyComponent message="&lt;3" />
 
 <MyComponent message={'<3'} />
 ```
 
-This behavior is usually not relevant. It's only mentioned here for completeness.
+Perilaku (*behavior*) biasanya tidak relevan. Dan hanya disebutkan di sini untuk melengkapi.
 
-### Props Default to "True" {#props-default-to-true}
+### Prop secara mendasar Diatur menjadi "True" {#props-default-to-true}
 
 If you pass no value for a prop, it defaults to `true`. These two JSX expressions are equivalent:
```

Comment:
Please translate this paragraph as well.

Reply:
>Jika Anda tidak mengoper nilai apapun ke sebuah *prop*, nilai *default*-nya adalah 'true'. Kedua ekspresi JSX ini adalah sama:

Sorry, it was my bad to leave this line untranslated
Thank you for your correction